skeleton/src/main/java/org/killbill/commons/skeleton/modules/StatsModule.java

```diff
@@ -80,9 +81,9 @@ protected void configure() {
         install(new AdminServletModule(healthCheckUri, metricsUri, pingUri, threadsUri));
 
         // Metrics/Jersey integration
-        final TimedResourceListener listener = new TimedResourceListener();
-        requestInjection(listener);
-        bindListener(Matchers.any(), listener);
+        final TimedResourceListener timedResourceTypeListener =
+                new TimedResourceListener(getProvider(GuiceContainer.class), getProvider(MetricRegistry.class));
```

Comment:
Nit: are you going on a new line to limit the code width? I think it makes diffs hard to read.